src/sql/orderbook/batch_rewards.sql

```diff
@@ -21,11 +22,12 @@ with hashed_observations as (SELECT block_number,
      reward_data as (select
                        -- observations
                        block_number,
-                       tx_hash,
+                       concat('0x', encode(tx_hash, 'hex'))  as tx_hash,
+                       concat('0x', encode(solver, 'hex'))  as solver,
                        execution_cost,
                        surplus,
                        fee,
-                       surplus + fee - reference_score as reward_eth,
+                       (surplus + fee - reference_score) / pow(10, 18) as reward_eth,
```

Comment:
We can keep this in WEI -- its just that we have had issues with precision and we were already doing this for COW. I can fix this since we do actually still have WEI. In that case we may have to cast this to `:: text`